src/BigQuery/Job.php

```diff
@@ -121,7 +127,8 @@ public function cancel(array $options = [])
     }
```

Comment:
Correct, it calls `jobs.getQueryResults` and uses the available hanging behavior with the default time of 10 seconds - if the results aren't available in that time frame it triggers a request to `jobs.getQueryResults` again on your behalf.